ajax/libs/goalProgress/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "goalProgress",
+  "filename": "goalProgress.min.js",
+  "version": "1.0",
+  "description": "Animated progress bar using jQuery to show how close you are to reaching your goal.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacious/goalProgress.git",
+    "homepage": "http://tinacious.github.io/goalProgress/"
+  },
+  "keywords": [
+    "goalProgress",
+    "tags",
+    "jQuery",
+    "animated"
+  ],
+  "author": "tinacious",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tinacious/goalProgress.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "goalProgress"
```

Comment:
Oh, Need to add `*` at the end of the word.
Let's modify from `"goalProgress"` to `"goalProgress*"`
This pattern can get the files.
Thank you.